Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL version="latest"
 ENV DEBIAN_FRONTEND noninteractive
 ENV APPDIR /app
 ENV LANG C.UTF-8
-ENV CONDA_VERSION 4.3.31
+ENV CONDA_VERSION 4.4.10
```

Comment:
`4.4.11` is the latest now https://github.com/conda/conda/blob/master/CHANGELOG.md#4411-2018-02-23

Reply:
4.4.10 is the latest miniconda release though